### ISSUE TITLE
Iranian Chain Stores

### ISSUE THREAD

**Issue by hadiomid** [osmlab/name-suggestion-index#6391]
shop=supermarket
brand=فروشگاه رفاه
brand:en=Refah Supermarket
brand:wikidata=Q7307085
brand:wikipedia=fa:فروشگاه_رفاه
name=فروشگاه رفاه
name:en=Refah Supermarket
opening_hours=08:00-23:30
phone=+98 21 54778
website=https://refah.ir/

[{"include":["ir"]}](https://location-conflation.com/?referrer=nsi&locationSet=%7B%22include%22%3A%5B%22ir%22%5D%7D)

**Comment by m-hue** [comment]
Hi @hdd064 - thanks for your contribution.
I have a couple of questions:
regarding Refah: 
- Please confirm the English name/brand should indeed be "Refah Supermarket", not just "Refah". 
- Do the phone number and opening hours actually apply to ALL their locations? The google translate version of the store listing here https://refah.ir/%D8%B4%D8%B9%D8%A8-%D8%B1%D9%81%D8%A7%D9%87/%D9%85%D8%B1%D8%A7%DA%A9%D8%B2-%D9%81%D8%B1%D9%88%D8%B4/ looks like both, the phone numbers and hours differ from one location to another.

regarding Ofoq Kourosh:
- a quick overpass search only shows one single location using the name you provided, however their website looks like they have  over 3,000 stores. Are there alternate spellings that should be considered? Or is this just extremely sparsely mapped currently?

Thank you!

**Comment by LaoshuBaby** [comment]
Besides, in #6481 all wikipedia tags are removed, and according to [multilingual name guideline](https://wiki.openstreetmap.org/wiki/Multilingual_names#Repeating_name_with_language_specific_tag) on OSM Wiki, It will be better if repeated Fars name again

**Comment by hdd064** [comment]
regarding Refah:

- Refah is Better. because called also Refah, Refah Market, and Refah Supermarket. But all of them are Refah.
- I added the Number of the Call Center of the Refah, But each Branch also has its own special Phone number.
- Yes, opening hours are different in some branches and can be removed.

regarding Ofoq Kourosh:
- You can search in Farsi to find Most of the branches = افق کوروش

**Comment by bhousel** [comment]
Yes we generally wouldn't add tags like `website`, `phone`, or `opening_hours`

(exception for a few brands that are well known for being `opening_hours=24/7`)
